common-util/src/main/kotlin/com/google/devtools/ksp/DescriptorUtils.kt

```diff
@@ -91,6 +91,7 @@ fun MemberDescriptor.toKSModifiers(): Set<Modifier> {
         DescriptorVisibilities.INTERNAL -> modifiers.add(Modifier.INTERNAL)
         // Since there is no modifier for package-private, use No modifier to tell if a symbol from binary is package private.
         JavaDescriptorVisibilities.PACKAGE_VISIBILITY, JavaDescriptorVisibilities.PROTECTED_STATIC_VISIBILITY -> Unit
+        DescriptorVisibilities.INVISIBLE_FAKE, DescriptorVisibilities.INVISIBLE_FROM_OTHER_MODULES -> Unit
```

Comment:
might not be the best option, per the comment above, no modifier is for handling package private, are these 2 visibilities considered package private? I would suggest to ask for a reproduce in the linked issue to understand what these visibilities are for. I did a brief look into the compiler, it seems that they are still consumed else where to determine the actual visibility.